packages/types/src/block-kit/blocks.ts

```diff
@@ -0,0 +1,89 @@
+// This file contains objects documented here: https://api.slack.com/reference/block-kit/blocks
+// TODO: go through https://api.slack.com/reference/block-kit/blocks and
+// - ensure JSdocs are up to date / added,
+// - define missing objects, and
+// - add further TODOs for future improvements / breaking changes, in prep for next major release
+
+import { PlainTextElement, MrkdwnElement } from './composition-objects';
+import { Button, Checkboxes, Datepicker, DateTimepicker, EmailInput, ImageElement, MultiSelect, NumberInput, Overflow, PlainTextInput, RadioButtons, Select, Timepicker, URLInput } from './elements';
+import { Action } from './extensions';
+
+export interface Block {
+  type: string;
+  block_id?: string;
+}
+
+export type KnownBlock = ImageBlock | ContextBlock | ActionsBlock | DividerBlock |
+SectionBlock | InputBlock | FileBlock | HeaderBlock | VideoBlock;
+
+export interface ActionsBlock extends Block {
```

Comment:
Seems worth it to adding additional annotation to more easily find the documentation link while we're at it? 

Fine to punt to the later TODO moment referenced in the comments above as well. 

```suggestion
// Reference: https://api.slack.com/reference/block-kit/blocks#actions
export interface ActionsBlock extends Block {
```

Reply:
Yes exactly, I will be doing exactly these kind of changes in the next pull request 😄 I didn't want to add more additional lines in this specific PR as the diff would be too confusing given the movement of code across files. I think that in the next PR, though, these kinds of documentation additions you suggest would be clearer and easier to review.